drone/build/build_start.go

```diff
@@ -19,6 +20,11 @@ var buildStartCmd = cli.Command{
 			Name:  "param, p",
 			Usage: "custom parameters to be injected into the job environment. Format: KEY=value",
 		},
+		cli.StringFlag{
+			Name:  "format",
+			Usage: "format output",
+			Value: tmplBuildInfo,
```

Comment:
only comment is that I think we are starting to use `Hidden: true` because the long template strings cause problems when you use `--help`